edx_proctoring/templates/proctored_exam/footer.html

```diff
@@ -10,13 +10,18 @@
 </div>
 
 <script type="text/javascript">
+  // Course waffle flag for verified name
+  var main = document.getElementById('main');
+  var isVerifiedNameEnabled = main.dataset.isVerifiedNameEnabled.toLowerCase() === 'true';
```

Comment:
Are these value set by waffle flag pipelined in from edx-platform?

Reply:
Yep, the course waffle flag is implemented here: https://github.com/edx/edx-platform/pull/28187